chapter_19_spiking_custom_auth.asciidoc

```diff
@@ -883,6 +886,7 @@ selenium.common.exceptions.NoSuchElementException: Message: Unable to locate
 element: input[name=email]; [...]
 [...]
 ----
+// JAN: I see the following exception: ModuleNotFoundError: No module named 'accounts', not the selenium one. We should remove all changes from settings.py/urls.py as well
```

Comment:
did you do the `git switch main`?  all the changes should be reverted at this point...